### PR TITLE
BZ1833393 - Expanding the subdomain delegation step in the 'Configuri…

### PR DESCRIPTION
**Pull request by pneedle-rh**
…ng Route53' procedure

Applies to master, branch/enterprise-4.3, branch/enterprise 4.4, branch/enterprise-4.5 and branch/enterprise-4.6.

This relates to https://bugzilla.redhat.com/show_bug.cgi?id=1833393.

Preview is at https://bz1833393--ocpdocs.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-account.html#installation-aws-route53_installing-aws-account

### CODE REVIEW

**Comment by pneedle-rh** [comment]
@openshift/team-documentation for peer review please. Please can someone also add 'Peer Review Needed', 'branch/enterprise-4.3', 'branch/enterprise-4.4', 'branch/enterprise-4.5', 'branch/enterprise-4.6' labels and 'Next Release' milestone? Thanks!

File: modules/installation-aws-route53.adoc [hunk 1]
```diff
@@ -44,5 +44,4 @@ uses. For example, if you registered your domain to a Route53 service in a
 different accounts, see the following topic in the AWS documentation:
 link:https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/domain-name-servers-glue-records.html#domain-name-servers-glue-records-procedure[Adding or Changing Name Servers or Glue Records].
 
-. If you use a subdomain, follow your company's procedures to add its delegation
-records to the parent domain.
+. If you are using a subdomain, add its delegation records to the parent domain. This gives Amazon Route53 responsibility for the subdomain. Follow the delegation procedure outlined by the parent domain's DNS provider. See link:https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/CreatingNewSubdomain.html[Creating a subdomain that uses Amazon Route 53 as the DNS service without migrating the parent domain] in the AWS documentation for an example high level procedure.
```
Comment (adellape) [inline modules/installation-aws-route53.adoc:47]: s/parent domain's DNS provider/DNS provider of the parent domain/

Per IBM SG about avoiding `’s` for possessive form of inanimate objects

**Review by adellape — COMMENTED** [review]
Minor nit but LGTM!

File: modules/installation-aws-route53.adoc (hunk 1 above)
Comment (pneedle-rh) [reply modules/installation-aws-route53.adoc]: Thank you Alex! Resolved.

**Comment by pneedle-rh** [comment]
@abhinavdahiya can you please review this from an SME perspective? Thanks!

**Review by abhinavdahiya — COMMENTED** [review]
LGTM

**Comment by pneedle-rh** [comment]
> LGTM

@abhinavdahiya thank you!

**Pull request merged**